integrated_channels/integrated_channel/exporters/content_metadata.py

```diff
@@ -209,39 +211,43 @@ def _check_matched_content_to_create(
                 integrated_channel_code=self.enterprise_configuration.channel_code(),
                 content_id=content_id,
             ).first()
-            incomplete_transmission = ContentMetadataItemTransmission.incomplete_create_transmissions(
```

Comment:
This lookup doesn't have to happen unless we've verified that the previous lookup (`past_deleted_transmission`) does not exist, so I moved it to after the `else`